res/database/Default/user/jorgaeff.xml

```diff
@@ -2020,9 +2020,9 @@
 			<!-- "Effekt: es stehen weniger Nachrichten zur Verfügung" -->
 		</news>
 
-		<news id="news-jorgaeff-lotto-01" type="0" creator="8936" created_by="jorgaeff">
+		<news id="news-jorgaeff-lotto-01" type="0" creator="8936" created_by="jorgaeff" comment="TODO: These never seem to appear. Check if changes to the availability script fixes this.">
 			<!-- "erscheint mittwochs ab 18.00 Uhr" -->
-			<availability script="TIME_WEEKDAY=2 &amp;&amp; TIME_HOUR >= 18" />
+			<availability script="TIME_WEEKDAY=2 &amp;&amp; TIME_HOUR &gt;= 18" />
```

Comment:
revert?

Reply:
reverted, although I wonder why you have to encode `&` but not `>`?

Reply:
"For historical reasons." In the "original" state of the database - when I started working on the project - the syntax was like this: make no unnecessary change, in particular if it makes reading the entry even harder.